src/components/AddTemplateModal.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<NcModal :key="newTemplate"
```

Comment:
This triggers a warning because a vue key can only be a string or a number. You can just get rid of the key.

Reply:
Done!